docs/_posts/2020-09-01-using-klee.md

```diff
@@ -64,8 +64,8 @@ The Rust compiler and KLEE are in the Dockerfile so start the Docker image
 by running
 
 ``` shell
-cd demos/simple/klee
```

Comment:
Both work - and the former is the way I tend to run it.

I think the existing order is better because it shows what you would do if verifying Oak, for example.

```
cd $HOME/oak # or wherever it is
$RVT_DIR/docker/run
# because we mount $PWD in docker, you are now in Oak and can access all the Oak files.
```

Reply:
I don't think the current one works. After you cd to klee docker/run is an invalid command. Am I missing something?